docs/using-the-compiler.rst

```diff
@@ -390,7 +390,9 @@ Input Description
         //   userdoc - User documentation (natspec)
         //   metadata - Metadata
         //   ir - Yul intermediate representation of the code before optimization
+        //   irAst - AST of Yul intermediate representation of the code before optimization in JSON format
         //   irOptimized - Intermediate representation after optimization
+        //   irOptimizedAsT - AST of intermediate representation after optimization in JSON format
```

Comment:
The `ast` output for Yul mode is still not documented.

Reply:
Do you mean when there is Yul input and the ast is requested? Reading the documentation, it doesn't seem to make the distinction for solidity input vs yul input. It simply says "ast - AST of all source files" which implies it works for yul source files as well. Do you think it is ok to be left like that?